composer.json

```diff
@@ -43,6 +43,10 @@
 		"scrutinizer/ocular": "^1.3",
 		"phpunit/phpunit": "4.*",
 		"squizlabs/php_codesniffer": "^2.5",
-        "php-mock/php-mock": "^1.0"
+        "php-mock/php-mock": "^1.0",
+		"symfony/event-dispatcher": "^3.0"
```

Comment:
It seem you have used tabs here. The indenting seems to be off.

Pro-Tip: `composer require --dev symfony/event-dispatcher`